front_end/panels/react_devtools/ReactDevToolsModel.ts

```diff
@@ -80,6 +80,10 @@ export class ReactDevToolsModel extends SDK.SDKModel.SDKModel<EventTypes> {
     window.addEventListener('beforeunload', () => this.#bridge?.shutdown());
   }
 
+  override dispose(): void {
+    this.#bridge?.removeListener('reloadAppForProfiling', this.#handleReloadAppForProfiling);
+  }
+
```

Comment:
We have some existing listeners that needs to be unsubscribed here too. Following up in https://github.com/facebookexperimental/rn-chrome-devtools-frontend/pull/129